src/python/pants/backend/python/util_rules/dists.py

```diff
@@ -118,6 +119,7 @@ class DistBuildRequest:
     input: Digest
     working_directory: str  # Relpath within the input digest.
     build_time_source_roots: tuple[str, ...]  # Source roots for 1st party build-time deps.
+    output_path: str  # Location of the output file within dist dir.
```

Comment:
Clarify in the comment that this is the location of the output *directory* within dist dir.

Reply:
Done.